guides/hardware/replacing-sd-card.md

```diff
@@ -7,4 +7,27 @@ tags:
 - moments
 ---
 
-## Goal
\ No newline at end of file
+## Goal: Accessing and Replacing the MicroSD Card
+You can increase Kuri’s local storage for moments with a few handy skills, a new MicroSD card, and a Phillips 0 screwdriver!
+
+The factory-installed MicroSD card which Kuri uses to store moments can hold about 16GB.  Follow the steps below to replace the existing card with a card that has more storage.
+
+### A few precautions:
+
+1. Kuri is not meant to be taken apart. You’ll need to remove some pieces on the underside of the robot to access the MicroSD card, so **please make sure to not access or remove the MicroSD card unnecessarily often.** If you access it once or twice, it should be fine!
+2. We have tested Sandisk MicroSD cards up to 256GB storage. We recommend using Sandisk Micro SD cards for maximum compatibility, but other cards should work well too.
+
+### Steps to access and replace Kuri’s SD Card:
+
+1. Make sure Kuri is on, then turn her off by holding down the power button on the back for 5 seconds until you hear Kuri going to sleep. Release the power button. The power button light will pulse. When it stops pulsing, the robot is off. Press the power button again to turn Kuri back on . Then turn her back on again. This way you have the most recent backup of the robot and don’t lose any data that is still on the existing MicroSD card.
```

Comment:
@tisho Do you have any idea why these instructions say "turn Kuri back on" twice in a row?

Reply:
My guess is a bad merge. I'd cut the rest of the paragraph after "Press the power button again to turn Kuri back on".